include/rangers-api/Application/Save/Accessors/OptionData.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+namespace app::save {
+    struct OptionData {
+
+    };
+    
+    class OptionAc : public SaveDataAccessor<OptionData> {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.